spec/ride_spec.rb

```diff
@@ -0,0 +1,78 @@
+require './lib/ride'
+
+class Ride
+  attr_reader :name,
+              :distance,
+              :loop,
+              :terrain
+
+  def initialize(name, distance, loop, terrain)
+    @name = name
+    @distance = distance
+    @loop = loop
+    @terrain = terrain 
+  end
+
+
+  it 'has a name' do
+    ride = Ride1.new('Walnut Creek Trail')
```

Comment:
`ride1 = Ride.new( # all necessary arguments go here, feel free to use interaction pattern for example)`
Be careful to not move on to the next test until you fix errors -- this would cause an error because our class name is Ride, not Ride1. We should also be passing the .new method a hash (check out the interaction pattern for an example)